ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -208,11 +208,8 @@ objects:
         fi
 
         # set up cloud-provider-specific env vars
-        export KUBE_SSH_BASTION="$( oc --insecure-skip-tls-verify get node -l node-role.kubernetes.io/master -o 'jsonpath={.items[0].status.addresses[?(@.type=="ExternalIP")].address}' ):22"
-        export KUBE_SSH_KEY_PATH=/tmp/cluster/ssh-privatekey
         if [[ "${CLUSTER_TYPE}" == "gcp" ]]; then
           export GOOGLE_APPLICATION_CREDENTIALS="${GCP_SHARED_CREDENTIALS_FILE}"
-          export KUBE_SSH_USER=cloud-user
           mkdir -p ~/.ssh
```

Comment:
what about this and then next one.

Reply:
We do use the .ssh directory and keys for the disaster recovery tests (https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L278-L295). If we ever want to run those on gcp, we'd still need the ssh private key

Reply:
Made me realize that we still need KUBE_SSH_KEYPATH

Reply:
Well I think in that case, let's just skip these ssh tests in openshift-tests and leave the template alone.

Reply:
Should we at least fix the user name for gcp?

Reply:
+1